src/Jackett.Common/Indexers/Torrentseeds.cs

```diff
@@ -146,7 +146,7 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
             searchUrl += "?" + queryCollection.GetQueryString();
             var response = await RequestStringWithCookiesAndRetry(searchUrl);
             var results = response.Content;
-            if (results.Contains("takelogin.php") || response.Cookies?.Contains("pass=deleted;") == true)
+            if (results.Contains("takelogin.php") || response.Cookies?.Contains("pass=-1") == true || response.IsRedirect && response.RedirectingTo.Contains("login.php"))
```

Comment:
I think it's enough with only one check `if (!results.Contains("/logout.php?")) => relogin` but I didn't test it.
If it works, do the same in the `ApplyConfiguration` function, you don't need to hardcode the full url `"https://torrentseeds.org/logout.php"`

I couldn't review the previous PR so, in single result search the torrent name ends with `... .​torrent` Could you fix that too?

Good work!

Reply:
Of course, will do.

Reply:
Your suggestion worked perfectly when I tested it.
`.torrent` is now quick-and-dirty just replaced.